frontend/src/tabs/zoom.pug

```diff
@@ -1,10 +1,17 @@
 #zoom
-  h1 {{ info.user.displayName }} ({{ info.user.name }})
-  a(v-on:click="openSummary") Show ramp chart for this period
-
   .zoom__title
-    .zoom__title--repo {{ info.user.repoName }} &nbsp;
-    .zoom__title--granularity.mini-font (granularity: one ramp per {{ filterTimeFrame }})
+    .zoom_title--group
+      h1(v-if="info.filterGroupSelection === 'groupByAuthors'") {{ info.user.displayName }} ({{ info.user.name }})
+      h1(v-else) {{ info.user.repoName }}
+    .zoom__title--chart
+      span &#8627; &nbsp;
+      h3(v-if="info.filterGroupSelection === 'groupByAuthors'") {{ info.user.repoName }}
+      h3(v-else) {{ info.user.displayName }} ({{ info.user.name }})
+    .zoom__title--period
+      span &#8627; &nbsp;
+      span {{ info.sinceDate }} ~ {{ info.untilDate }} &nbsp;
+      a(v-on:click="openSummary") [Show ramp chart for this period]
```

Comment:
Is it possible to make changes to the sinceDate and untilDate instead?

Reason: The dotted selection get reset or not in the correct range of dates.

Reply:
Sorry but I don't really get what you mean. Do u mean making changes to the sinceDate and untilDate when clicking the `show ramp chart for this period` button?

Reply:
Yes. As using the dotted selection is not reliable.

Reply:
Oh yes that issue is being solved in #847. `sinceDate` and `untilDate` of zoom will be used instead of the dotted selections.